src/app/ui/editor/pixels_movement.cpp

```diff
@@ -281,6 +281,8 @@ void PixelsMovement::setTransformationBase(const Transformation& t)
   }
 
   // If "fullBounds" is empty is because the cel was not moved
+  if (m_site.tilemapMode() == TilemapMode::Tiles)
+    fullBounds = m_site.grid().alignBounds(fullBounds);
```

Comment:
This part is useful to correctly 'invalidate' editor regions if the selection started in Pixels mode and the selection bounds doesn't match with the tilemap grid. I can show you an animation if needed.

Reply:
There is no need of an animation, but it would be great if you can include a comment in the code why each part is required (and in the commit message include all the fixes included in the commit).